.eslintrc.js

```diff
@@ -60,6 +59,19 @@ module.exports = {
       parserOptions: {
         project: './tsconfig.json',
       },
+      rules: {
+        'prefer-arrow/prefer-arrow-functions': 'error',
+        'react/function-component-definition': [
+          'error',
+          {
+            namedComponents: 'arrow-function',
+            unnamedComponents: 'arrow-function',
+          },
+        ],
+        'arrow-body-style': 'error',
+        'react/require-default-props': 'off',
+        'react/prop-types': 'off',
+      },
```

Comment:
1. Why did you add these rules to overrides instead of applying them to all the files?
2. Can you explain why do you prefer using these rules? Why should we use the bottom two when they forgo type safety?

Reply:
1. That's a good question. I've tried, but some of the rules are later overridden by other plugins. I can put the rules into a variable and put it in both places. Or do you have a better suggestion?
2. The bottom two are not useful with Typescript.

Reply:
1. Yes, let's create `commonRules` and do the some thing as with `commonPresets`.
2. OK. But what about the arrow functions? Why do you prefer them?

Reply:
I've created #9 for point 1, as it's currently broken in `main` too. Merge it into this branch after it's merged into `main`.

Reply:
> OK. But what about the arrow functions? Why do you prefer them?

It is a stylistic preference. I'm used to using arrow functions.